test/specs/controller.line.tests.js

```diff
@@ -964,4 +964,60 @@ describe('Chart.controllers.line', function() {
     expect(isNaN(x)).toBe(false);
     expect(isNaN(y)).toBe(false);
   });
+
+  it('should honor spangap interval forwards', function() {
+    var chart = window.acquireChart({
+      type: 'line',
+      data: {
+        datasets: [{
+          xAxisKey: 'x',
+          yAxisKey: 'y',
```

Comment:
Do these lines affect the tests in any way? These should be the defaults.

Reply:
No, they don't. I added them when what I really needed was the linear scale, and forgot to remove them. I went ahead and pushed an update.